_hackbars/yashikacac.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Open Source
+author: yashikacac
+---
+
+Help climate grow this hacktober fest using your developer skills!
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```

Comment:
hey @yashikacac, Add your message on `climate change` and make a commit

Reply:
sure

Reply:
Thanks

Reply:
May you please let me know whether is it done or not.